src/witan/models/dem/ccm/fert/fertility.clj

```diff
@@ -104,28 +103,74 @@
          (calc-scaled-fert-rates asfr-birth-data-year)
          (ds/select-columns [:gss-code :sex :age :year :fert-rate]))}))
 
-(defworkflowfn project-asfr-finalyearhist-fixed
-  "Takes dataset of historic age specific fertility rates, and parameter
-   for the base year of fertility data. Returns dataset with projected
-   age specific fertility rates, calculated using the jumpoff year average
-   method (see docs)."
-  {:witan/name :ccm-fert/project-asfr-finalyearhist-fixed
+(defworkflowfn project-asfr-1-0-0
```

Comment:
no reason to keep this fn as far as I am concerned as functionality captured in 1-1-0